strawberryfields/compilers/xunitary.py

```diff
@@ -22,21 +24,25 @@
 
 import strawberryfields.ops as ops
 
-from .circuit_specs import CircuitSpecs, Ranges
-from .gbs import GBSSpecs
+from .compiler import Compiler
+from .gbs import GBS
 from .gaussian_unitary import GaussianUnitary
 
 
-class Xunitary(CircuitSpecs):
-    """Circuit specifications for the X class of circuits."""
+def list_duplicates(seq):
+    """Returns a dictionary representing the duplicated values in the sequence
```

Comment:
(Minor suggestion): a generator object seems to be returned here
```suggestion
    """Returns a generator representing the duplicated values in the sequence
```